ecpy_hqc_legacy/tasks/tasks/instr/views/dc_views.enaml

```diff
@@ -1,30 +1,29 @@
 # -*- coding: utf-8 -*-
 # -----------------------------------------------------------------------------
-# Copyright 2015-2016 by EcpyHqcLegacy Authors, see AUTHORS for more details.
+# Copyright 2016-2017 by EcpyHqcLegacy Authors, see AUTHORS for more details.
 #
 # Distributed under the terms of the BSD license.
 #
 # The full license is in the file LICENCE, distributed with this software.
 # -----------------------------------------------------------------------------
-"""Views for the DC sources related tasks.
+"""Views for the microwave sources related tasks.
```

Comment:
Why ?

Reply:
I have no idea why I would have changed this. Makes no sense. Changing it back.

Reply:
I see, I copied this file from the rf views, and forgot to change microwave to DC